src/sentry/static/sentry/app/components/events/eventDataSection.tsx

```diff
@@ -159,7 +159,7 @@ const SectionHeader = styled('div')<{isCentered?: boolean}>`
 
     text-transform: none;
   }
-  & small a {
```

Comment:
Is `CrashHeader` the only thing that uses this css?

Reply:
Yep. I did a search on `<small>` in `src/sentry/static/sentry/app/components/events/*`.